src/Nethermind/Nethermind.Facade/BlockchainBridge.cs

```diff
@@ -281,6 +324,117 @@ public CallOutput CreateAccessList(BlockHeader header, Transaction tx, Cancellat
             transactionProcessor.CallAndRestore(transaction, callHeader, tracer);
         }
 
+        private (bool Success, string Error) TryMultiCallTrace(BlockHeader parent, MultiCallBlockStateCallsModel[] blocks,
```

Comment:
I would move this whole method to some helper class to unclutter BlockchainBridge

Reply:
Helper class introduced